src/utils/hastebinize.js

```diff
@@ -0,0 +1,22 @@
+const fetch = require('node-fetch');
+
+// Will turn `text` into a hastebin document. 
+// Should return a Hastebin URL.
+
+async function hastebinize(text) { // Message.reply_to_message.text as argument.
+  try {
+    const requestToHastebin = await fetch('https://hastebin.com/documents', { // POSTs message to hastebin. Should return an object
+      method: 'POST',                                                         // that contains hastebin document id.
+      body: text                                                              // Example: {key: 'ezuhusunaq'}
+    })
+
+    const response = await requestToHastebin.json()
+
+    return `https://hastebin.com/${response?.key}`
```

Comment:
No idea what this syntax is, did you mean `response.key` or am I missing some javascript knowledge?

Reply:
It's new syntax feature recently added in ES2020. See:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining